iscript/src/iscript/mac.py

```diff
@@ -992,20 +992,22 @@ async def create_pkg_files(config, sign_config, all_paths, requirements_plist_pa
             asyncio.ensure_future(
                 semaphore_wrapper(
                     semaphore,
-                    run_command(
-                        [
-                            "pkgbuild",
-                            "--install-location",
-                            "/Applications",
-                        ]
-                        + cmd_opts
-                        + [
-                            "--component",
-                            app.app_path,
-                            app.tmp_pkg_path1,
-                        ],
-                        cwd=app.parent_dir,
-                        exception=IScriptError,
+                    retry_async(
+                        func=run_command,
+                        kwargs={
+                            "cmd": [
```

Comment:
We might want `cmd` to be in `"args": ("pkgbuild", ...),`

Reply:
Same difference, no?

Reply:
Now that I think about it, having it in `args`, makes it more confusing.
We want it to be:
```
run_command(["pkgbuild", "--install-location"....], cwd="foo", exception=Bar)
```
But having `"args": ("pkgbuild", "--install-location"....),` means we'd be calling it as:
```
run_command("pkgbuild", "--install-location"...., cwd="foo", exception=Bar)
```
Which isn't the same, (but could be solved by double wrapping the args) so in order to keep the code more readable, I'd vote for using just kwargs.

Reply:
These are supposed to essentially mirror `*args` and `**kwargs` https://www.geeksforgeeks.org/args-kwargs-python/

Reply:
and yeah, I suppose it would be `"args": (["pkgbuild", ...])`